Sources/qiskit/datastructures/String+Base64.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 IBM RESEARCH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+import Foundation
+
+extension String {
+    func base64Decoded() -> String? {
```

Comment:
Used to decode the javascript libraries [here](https://github.com/indisoluble/qiskit-sdk-swift/blob/master/Sources/qiskit/tools/visualization/charts/BarChartHtmlFactory.swift#L37) & [here](https://github.com/indisoluble/qiskit-sdk-swift/blob/master/Sources/qiskit/tools/visualization/charts/StateCityChartHtmlFactory.swift#L42).